examples/replication/variables.tf

```diff
@@ -13,24 +13,15 @@ variable "prefix" {
 variable "resource_tags" {
   type        = list(string)
   description = "Optional list of tags to be added to created resources"
-  default     = []
 }
 
 # region needs to provide cross region support.
 variable "region" {
   description = "Region where resources will be created"
   type        = string
-  default     = "us-south"
 }
 
 variable "resource_group" {
   type        = string
   description = "An existing resource group name to use for this example, if unset a new resource group will be created"
-  default     = null
```

Comment:
leave `null` as default value because `resource_group` is not a required variable. If `null` is passed, the example will create a new resource group